centreon/www/include/options/media/images/syncDir.php

```diff
@@ -191,13 +190,23 @@ function checkPicture(
             int $directoryId,
             CentreonDB $pearDB
     ): void {
-        global $allowedExt, $regCounter, $gdCounter;
+        global $regCounter, $gdCounter;
+
+        $mimeTypeFileExtensionConcordance = [
+            "jpg" => "image/jpeg",
+            "jpeg" => "image/jpeg",
+            "gif" => "image/gif",
+            "png" => "image/png",
+            "gd2" => "",
+        ];
```

Comment:
syncDir does not handle SVGs ?  It is possible to upload SVG so I think the sync should also manage it, or I missing something else ?